src/predict.py

```diff
@@ -1,5 +1,25 @@
 import numpy as np
 from skimage import measure
+from . import config as cf
+from pathlib import Path
+import gdown
+
+from fastai.vision import load_learner
+
+
+def load_learn(path=Path(__file__).parents[1], model=cf.MODEL):
```

Comment:
Perhaps `get_learner` or `fetch_learner` or `download_learner` would be a better name. The current one is kind of confusing with the fastai `load_learner`.